src/lib/Scenes/Collection/Collection.tsx

```diff
@@ -25,8 +28,11 @@ interface CollectionState {
 export class Collection extends Component<CollectionProps, CollectionState> {
   state = {
     sections: [],
+    isArtworkGridVisible: false,
+  }
+  viewabilityConfig = {
+    viewAreaCoveragePercentThreshold: 5,
```

Comment:
Sooo in plain English this means the artwork grid must cover 5% of the viewport before it is considered visible? (I _think_ so, but it was a little hard to tell from the gif bc of that yellow alert.)

Reply:
I _think_ so. [I found the docs](https://facebook.github.io/react-native/docs/flatlist#viewareacoveragepercentthreshold) and it appears to be the case. Seems like we want the filter button to appear only when this is the case, which makes sense 👍 In any case, a short comment would be super-helpful here!